app/controllers/DashboardController.scala

```diff
@@ -29,26 +29,27 @@ import uk.gov.hmrc.play.http.HeaderCarrier
 import utils.{GatekeeperAuthProvider, GatekeeperAuthWrapper}
 import views.html.approvedApplication._
 import views.html.dashboard._
+import views.html.developers.developers
 import views.html.review._
 
 import scala.concurrent.Future
 
 object DashboardController extends DashboardController {
+  override val applicationConnector = ApplicationConnector
+  override val developerConnector: DeveloperConnector = DeveloperConnector
+
   override def authProvider = GatekeeperAuthProvider
 
   override def authConnector = AuthConnector
-
-  override val applicationConnector = ApplicationConnector
-  override val developerConnector: DeveloperConnector = DeveloperConnector
 }
 
 trait DashboardController extends FrontendController with GatekeeperAuthWrapper {
 
-  implicit def dateTimeOrdering: Ordering[DateTime] = Ordering.fromLessThan(_ isBefore _)
-
   val applicationConnector: ApplicationConnector
   val developerConnector: DeveloperConnector
 
+  implicit def dateTimeOrdering: Ordering[DateTime] = Ordering.fromLessThan(_ isBefore _)
```

Comment:
Not 100% sure why this was ever a `def` in the first place - it would be as `val`uable as a `val`.